src/gandalf.ts

```diff
@@ -68,6 +65,10 @@ class Gandalf extends React.Component<{}, GandalfState> {
       return formValues;
     }, {});
   }
+
+  areFormFieldsReady(): boolean {
```

Comment:
They're referred to as "fields" elsewhere, so it's unusual to see them referred to as "form fields" here.

Reply:
So are you suggesting something like `areFieldsReady()`?

Reply:
That would be more consistent, right?